content/blog/tidymodels-2022-q3/index.Rmd

```diff
@@ -0,0 +1,248 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2022-q3
+title: "Q3 2022 tidymodels digest"
+date: 2022-10-20
+author: Max Kuhn
+description: >
+    Our post-RStudio conference productivity has been high! This post talks about tidymodels updates from the last few months. 
+photo:
+  url: https://unsplash.com/photos/PyDaL4PcLoQ
+  author:  Simon Spieske
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [roundup] 
+tags: [tidymodels, agua, recipes, h2o]
+---
+
+
+```{r}
+#| include: false
+#| label: startup
+
+library(tidymodels)
+library(agua)
+library(finetune)
+library(doMC)
+registerDoMC(cores = 10)
+
+tidymodels_prefer()
+theme_set(theme_bw())
+options(pillar.advice = FALSE, pillar.min_title_chars = Inf, width = 120)
+```
+```{r}
+#| label: get-repo-info
+#| include: FALSE
+
+since <- "2022-07-19"
+
+source("repo-functions.R")
+
+
+tm_data <- 
+  map_dfr(tm_pkgs, get_current_release) %>% 
+  filter(date > ymd(since)) %>% 
+  mutate(
+    repo = paste0("tidymodels/", package),
+    thanks = map_chr(repo, return_tidy_thanks, from = since),
+    thanks = glue("- {package}: {thanks}"),
+    news = glue("- {package} [({version})](https://{package}.tidymodels.org/news/index.html)")
+  )
+
+txt_pkg_list <- knitr::combine_words(tm_data$package)
+```
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. 
+
+Since the beginning of 2021, we have been publishing [quarterly updates](https://www.tidyverse.org/categories/roundup/) here on the tidyverse blog summarizing what's new in the tidymodels ecosystem. The purpose of these regular posts is to share useful new features and any updates you may have missed. You can check out the [`tidymodels` tag](https://www.tidyverse.org/tags/tidymodels/) to find all tidymodels blog posts here, including our roundup posts as well as those that are more focused, like these from the past month or so:
+
+- [Improvements to model specification checking in tidymodels](https://www.tidyverse.org/blog/2022/10/parsnip-checking-1-0-2/)
+- [brulee 0.2.0](https://www.tidyverse.org/blog/2022/09/brulee-0-2-0/)
+- [Announcing bundle](https://www.tidyverse.org/blog/2022/09/bundle-0-1-0/)
+- [censored 0.1.0](https://www.tidyverse.org/blog/2022/08/censored-0-1-0/)
+- [rsample 1.1.0](https://www.tidyverse.org/blog/2022/08/rsample-1-1-0/)
+
+Since [our last roundup post](https://www.tidyverse.org/blog/2022/07/tidymodels-2022-q2/), there have been CRAN releases of `r nrow(tm_data)` tidymodels packages. Here are links to their NEWS files:
+
+```{r}
+#| echo: FALSE
+#| results: asis
+
+cat(tm_data$news, sep = "\n")
```

Comment:
Love this.

Reply:
Yeah. I can't imagine doing it manually.